src/actions/sources/select.js

```diff
@@ -113,13 +115,7 @@ export function selectLocation(location: Location) {
     const source = sourceRecord.toJS();
 
     dispatch(addTab(source, 0));
-    dispatch(
-      ({
-        type: "SELECT_SOURCE",
-        source,
-        location
-      }: Action)
-    );
+    dispatch(setSelectedLocation(source, location));
```

Comment:
feels like an appropriate name and avoids a naming conflict with existing `selectSource`